app/src/main/java/com/example/plan2meet/dao/IContactsDAO.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.plan2meet.dao
+
+import com.example.plan2meet.dto.Contacts
+import retrofit2.Call
+
+interface IContactsDAO {
+    fun getAllContacts() : Call<ArrayList<Contacts>>
```

Comment:
Is this an actual Retrofit call you plan to make?  If so, it's missing the @GET endpoint.